genesyscloud/routing_queue/resource_genesyscloud_routing_queue.go

```diff
@@ -23,116 +23,10 @@ import (
-			},
-			"enable_auto_answer": {
-				Description: "Auto-Answer for digital channels(Email, Message)",
-				Type:        schema.TypeBool,
-				Optional:    true,
-				Default:     false,
-			},
-			"service_level_percentage": {
-				Description:  "The desired Service Level. A float value between 0 and 1.",
-				Type:         schema.TypeFloat,
-				Required:     true,
-				ValidateFunc: validation.FloatBetween(0, 1),
-			},
-			"service_level_duration_ms": {
-				Description:  "Service Level target in milliseconds. Must be >= 1000",
-				Type:         schema.TypeInt,
-				Required:     true,
-				ValidateFunc: validation.IntAtLeast(1000),
-			},
-		},
-	}
-
-	queueMemberResource = &schema.Resource{
-		Schema: map[string]*schema.Schema{
-			"user_id": {
-				Description: "User ID",
-				Type:        schema.TypeString,
-				Required:    true,
-			},
-			"ring_num": {
-				Description:  "Ring number between 1 and 6 for this user in the queue.",
-				Type:         schema.TypeInt,
-				Optional:     true,
-				Default:      1,
-				ValidateFunc: validation.IntBetween(1, 6),
-			},
-		},
-	}
-
-	directRoutingResource = &schema.Resource{
-		Schema: map[string]*schema.Schema{
-			"backup_queue_id": {
-				Description: "Direct Routing default backup queue id (if none supplied this queue will be used as backup).",
-				Type:        schema.TypeString,
-				Optional:    true,
-				Computed:    true,
-			},
-			"agent_wait_seconds": {
-				Description: "The queue default time a Direct Routing interaction will wait for an agent before it goes to configured backup.",
-				Type:        schema.TypeInt,
-				Optional:    true,
-				Default:     60,
-			},
-			"wait_for_agent": {
-				Description: "Boolean indicating if Direct Routing interactions should wait for the targeted agent by default.",
-				Type:        schema.TypeBool,
-				Optional:    true,
-				Default:     false,
-			},
-			"call_use_agent_address_outbound": {
-				Description: "Boolean indicating if user Direct Routing addresses should be used outbound on behalf of queue in place of Queue address for calls.",
-				Type:        schema.TypeBool,
-				Optional:    true,
-				Default:     true,
-			},
-			"email_use_agent_address_outbound": {
-				Description: "Boolean indicating if user Direct Routing addresses should be used outbound on behalf of queue in place of Queue address for emails.",
-				Type:        schema.TypeBool,
-				Optional:    true,
-				Default:     true,
-			},
-			"message_use_agent_address_outbound": {
-				Description: "Boolean indicating if user Direct Routing addresses should be used outbound on behalf of queue in place of Queue address for messages.",
-				Type:        schema.TypeBool,
-				Optional:    true,
-				Default:     true,
-			},
-		},
-	}
-)
+var bullseyeExpansionTypeTimeout = "TIMEOUT_SECONDS"
 
 func getAllRoutingQueues(_ context.Context, clientConfig *platformclientv2.Configuration) (resourceExporter.ResourceIDMetaMap, diag.Diagnostics) {
```

Comment:
can you add a proxy for this

Reply:
Even though I put routing queues into it's own package, I was not doing a refactor. Doing a full refactor would take a long time and there's still a ticket for that. But since I was touching routing_queues I thought I would put it into it's own package to help speed things up for when we are refactoring it later.